src/app/core/component/graph-display/graph-display.component.html

```diff
@@ -74,35 +74,35 @@ <h4 style="margin: 1em">Relations of {{currentPattern?.name}}</h4>
   <mat-sidenav-content>
     <network-graph #graphWrapper
                    (onAddedEdge)="edgeAdded($event)"
+                   (onRemovedEdge)="edgeRemoved($event)"
                    (onClickedBackground)="backgroundClicked()"
                    (onClickedNode)="nodeClicked($event)"
-                   (onNodePositionChange)="saveGraph()"
+                   (onNodePositionChange)="nodePositionChanged($event)"
                    [ppGraphEventEmmiter]
                    classes=" highlighted-edge low-opacity-node low-opacity-edge"
                    mode="layout"
                    style="width: 100%; height: 100%"
                    zoom="both">
-      <style slot="style">
-        svg {
-          width: 1000px;
-          height: 700px;
-        }
-      </style>
       <svg #svg slot="graph" [class.left-sidenav-open]="!!patternContainer">
```

Comment:
Is !!patternContainer intended?

Reply:
Yes, it checks if there is an object in the variable `patternContainer` and iff it evaluates to `true`.